src/odemis/acq/fastem.py

```diff
@@ -126,10 +131,14 @@ def _calculate_field_indices(self):
         field_res = self._multibeam.resolution.value
 
         # The size of a field consists of the effective cell images excluding overscanned pixels.
-        field_size = (field_res[0] * px_size[0], field_res[1] * px_size[1])
+        field_size = (field_res[0] * px_size[0],
+                      field_res[1] * px_size[1])
+
         # Note: floating point errors here, can result in an additional row or column of fields (that's fine)
-        n_hor_fields = math.ceil(abs(r - l) / field_size[0])
-        n_vert_fields = math.ceil(abs(b - t) / field_size[1])
+        # When fields overlap the number of fields is calculated by subtracting the size of the field that overlaps
```

Comment:
How did you get to this formula? It produces the correct values, but I am curious as it is not really intuitive :) .

Reply:
Can you add this example as a comment? I think the next reader might also struggle here.